Documents/Management/WeeklyReports/mainWeeklyReport.md

```diff
@@ -269,6 +269,27 @@ The challenges encountered this week are still the same as the previous week but
 
 The fifth week was short, the team finished simulating the functionality of each component with LEDs. The team also managed to finish the documentation like the test plan and the user manual. The team is still motivated to finish the project with the presentation next week.
 
+## Week 6 (15-19 April 2024)
+
+The sixth week was for the preparation of the oral on Friday.
```

Comment:
"oral presentation" please cause their is lot of things with "oral" and some of them can be weird.

![](https://media0.giphy.com/media/QBM7uBQAzLxLczIDEN/200w.gif?cid=6c09b9523sq4x2zndwdydoqyqztgric2qn24p74avy9etnbl&ep=v1_gifs_search&rid=200w.gif&ct=g)

Reply:
![](https://media1.tenor.com/m/B7TP3JU7qcAAAAAC/fixed.gif)